docs/reference/security.rst

```diff
@@ -348,20 +350,14 @@ The following configuration for the SonataUserBundle defines:
             #sonata.admin.security.mask.builder:
```

Comment:
Can you take a look at this PR and modify things here if neccessary? https://github.com/sonata-project/SonataAdminBundle/pull/4962